From 370301cf4cbb8123fdf03f757f3e931d85e2cefe Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Sat, 21 Feb 2026 19:46:29 +0100
Subject: [PATCH] COMP: Add Annotations module MRML include directory

Add vtkSlicerAnnotationsModuleMRML include directories to Markups MRML
CMakeLists to enable access to Annotations module headers.
---
 Modules/Loadable/Markups/MRML/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Modules/Loadable/Markups/MRML/CMakeLists.txt b/Modules/Loadable/Markups/MRML/CMakeLists.txt
index f21707b791..487c67676c 100644
--- a/Modules/Loadable/Markups/MRML/CMakeLists.txt
+++ b/Modules/Loadable/Markups/MRML/CMakeLists.txt
@@ -10,6 +10,7 @@ set(${KIT}_INCLUDE_DIRECTORIES
   ${RapidJSON_INCLUDE_DIR}
   ${vtkSlicer${MODULE_NAME}ModuleVTKWidgets_SOURCE_DIR}
   ${vtkSlicer${MODULE_NAME}ModuleVTKWidgets_BINARY_DIR}
+  ${vtkSlicerAnnotationsModuleMRML_INCLUDE_DIRS}
   )
 
 set(${KIT}_SRCS
-- 
2.52.0

